doc/downgrade.8.md

```diff
@@ -60,20 +60,34 @@ The columns have the following meaning:
 
 # OPTIONS
 
-## DOWNGRADE OPTIONS
+## DOWNGRADE OPTIONS WITH ARGUMENTS
```

Comment:
FWIW, I don't it particularly valuable to have distinct sections.

Reply:
Will revert back.